tests/app/test_retriever.py

```diff
@@ -13,15 +13,15 @@
 from secure_message.constants import MESSAGE_QUERY_LIMIT
 from secure_message.services.service_toggles import party
 from secure_message.validation.user import User
-from tests.app import test_utilities
-from tests.app.test_utilities import BRES_SURVEY, get_args
+from tests.app.test_utilities import get_args
 
 
 class RetrieverTestCaseHelper:
 
     default_internal_actor = 'internal_actor'
     second_internal_actor = 'second_internal_actor'
     default_external_actor = 'external_actor'
+    BRES_SURVEY = "33333333-22222-3333-4444-88dc018a1a4c"
 
```

Comment:
Should we have a `setUp` function here prior to remove the unresolved ref to all the engine refs in this class similar to how it is done in the `test_modifier.py` class?

Reply:
Doing this would require a complete restructuring of the tests.  Considering how long this PR has gone on for and how this PR actually tidied this bit up from what it was, it should be done in some other PR.